### DISCUSSION TITLE
[Script request] Paper Merge DMS

### DISCUSSION THREAD

**Discussion by ellic052** [community-scripts/ProxmoxVE#705] [Request script]
### Application Name

Paper Merge 3.x

### Website

https://docs.papermerge.io/3.2/

### Description

Papermerge is a open source document management system designed to work with scanned documents (also called digital archives). It extracts text from your scans using OCR, indexes them, and prepares them for full text search. Papermerge provides look and feel of modern desktop file browsers. It has features like dual panel document browser, drag and drop, tags, hierarchical folders and full text search so that you can efficiently store and organize your documents.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Node815** [comment]
+1  It would be a great addon!

**Comment by Najdat** [comment]
Isn't that exactly what Paperless-NGX does?

**Reply by Node815** [reply]
Papermerge gives you a different way of organizing.  Where Paperless-NGX organizes by tabs, Papermerge gives you folders which for some make it an easier way of organizing documents instead of tagging them.   Both have their strengths so it's a matter of choice. :)
